project3/coordinator/messenger.h

```diff
@@ -53,26 +48,37 @@ class Messenger {
    * participant.
    * @return true on success, false on failure.
    */
-  bool SendMissedMessages(const MessageLog::Timestamp &reconnection_time);
+  bool SendMissedMessages(const MessageLog::Timestamp& reconnection_time);
+
+  /**
+   * @brief Getter for this messenger's participant.
+   * @return This messenger's participant.
+   */
+  ConnectedParticipants::Participant GetParticipant() const;
 
  private:
-  /// The outgoing message queue.
-  std::shared_ptr < queue::Queue<MessageLog::Message>> msg_queue_;
+  /// == operator overload.
+  bool operator==(const Messenger&) const;
```

Comment:
The fact that this compiles even though this is private suggests that it's not necessary.